docs/configuration.md

```diff
@@ -79,6 +79,12 @@ devices:
     alias: <alias>
     connection_type: <connection_type>
     device_type: <device_type>
+# TCPIP connection
+  - address: <ip_address_or_hostname>
+    alias: <alias>
+    connection_type: TCPIP
+    device_type: <device_type>
+    lan_device_name: <lan_device_name>
 # usb connection
```

Comment:
```suggestion
# USB connection
```

Reply:
Probably should update all comments to have proper capitalization.

Reply:
change has been pushed.